static/src/stylesheets/module/facia/item-tones/_tone-live.scss

```diff
@@ -80,3 +80,53 @@
 .tone-live--most-popular .fc-item__kicker {
     color: colour(live-accent);
 }
+
+.fc-item__liveblog-blocks {
```

Comment:
all these styles are global, not behind any `live` tone restriction – should they be?